docs/content/manual/upgrade/upgrade-from-traditional-network-design.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Upgrade Harvester from traditonal cluster network design (before v1.1.0)
+---
+
+
+## Category: 
+* Upgrade Harvester
+
+## Environment requirement
+1. Network environment have available VLAN id setup on DHCP server can be assigned to Harvester
```

Comment:
See previous comment about this line

Reply:
Thanks for the check, I modified the line 10 of Environment requirement to better understanding 

1. Network environment has available VLAN id setup on DHCP server
1. DHCP server has setup the IP range can allocate to above VLAN id
1. Harvester node can route to DHCP server through VLAN id to retrieve IP address